editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -687,6 +683,22 @@ export function convertGroupToFrameCommands(
   ]
 }
 
+export type GroupChildElement = JSXElementLike | JSXConditionalExpression
+
+export function elementCanBeAGroupChild(
+  element: JSXElementChild | null,
+): element is GroupChildElement {
+  if (element == null) {
+    return false
+  }
+  return (
+    isJSXElementLike(element) ||
+    (isJSXConditionalExpression(element) &&
+      // do not allow grouping zero-sized conditionals
+      !(isNullJSXAttributeValue(element.whenTrue) && isNullJSXAttributeValue(element.whenFalse)))
```

Comment:
Ooooh this is a tricky one. So this is only checking that one side of the conditional is not null, but isn't checking that the active branch is not null. Even if we were to check that the active branch is not null, that doesn't protect us from the active branch being switched. Why do we actually need the not-null check? I'm wondering if it would be better to support null children now 🤔

Reply:
heh you're right and this should use the active branch check actually. This is not a requirement tho, it's just an additional defensive condition I added deliberately, but nothing is preventing from having zero-sized conditionals in groups (they will just look as invalid in the navigator). Shall I just remove the condition and call it a day?

Reply:
If that doesn't break anything then yeah you might as well